docs/src/main/asciidoc/security-openid-connect-multitenancy.adoc

```diff
@@ -149,7 +149,7 @@ public class CustomTenantResolver implements TenantResolver {
 From the implementation above, tenants are resolved from the request path so that in case no tenant could be inferred, `null` is returned to indicate that the default tenant configuration should be used.
 
 [NOTE]
-===
+====
```

Comment:
The closing one is also three equal signs. And I can see that others in the same doc have the same issue. Could you  fix all of them? Thanks!

Reply:
Sorry, I fixed it in the next commit